test/kickstart-templates/includes/main-ostreecontainer.cfg

```diff
@@ -0,0 +1,10 @@
+# Mount the bootc image directory from the hypervisor
+%pre-install --log=/dev/console --erroronfail
+
+mkdir -p /mnt/bootc-images
+mount -v -t virtiofs bootc-images /mnt/bootc-images
```

Comment:
Can you remind what was the problem with the registry? I think we should add registry pretty soon